src/test/java/bisq/price/spot/providers/QuoineTest.java

```diff
@@ -27,7 +27,8 @@ public class QuoineTest extends AbstractExchangeRateProviderTest {
 
     @Test
     public void doGet_successfulCall() {
-        doGet_successfulCall(new Quoine(new StandardEnvironment()));
+//        not compiling
```

Comment:
Best to rebase your branch on https://github.com/bisq-network/bisq-pricenode/pull/14 which contains the fixes for the test issues.

Reply:
I will, thanks!